docs/source/references/configurations.rst

```diff
@@ -70,4 +70,49 @@ It only makes sense to reset the session when the backend database is changed si
 Thus, the only place to change the LineaPy database is at the beginning of the notebook.
 
 Note that, you need to make sure whenever you are setting `LINEAPY_DATABASE_URL`, you point to the  `LINEAPY_ARTIFACT_STORAGE_DIR`.
-If not, ``Artifact.get_value`` might return an error that is related cannot find underlying pickle object. 
\ No newline at end of file
+If not, ``Artifact.get_value`` might return an error that is related cannot find underlying pickle object.
+
+
+
+Artifact Storage Location
+=========================
+
+You can change the artifact storage location by setting the `LINEAPY_ARTIFACT_STORAGE_DIR` environmental variable, or other ways mentioned in the above session.
+LineaPy is using the IO handler from ``pandas`` directly.
+In theory, we can use various filesystems like S3, GCS, Azure, etc... to store the artifact, just like we are using ``pandas.DataFrame.to_csv`` to save CSV files.
+Note that, LineaPy only supports S3 at this moment.
```

Comment:
I dont think this is true anymore. We support S3 only in the sense that you can install s3fs etc using lineapy[s3]. but beyond that if the user manually installs all the necessary libs listed in fsspec / universal_pathlib, we essentially support all of those backends and do not need to single out S3 here.

Reply:
Just want to make sure we test the filesystem before saying we support them.